widgets/Split/index.tsx

```diff
@@ -111,7 +113,7 @@ export const Split = ({
   }, []);
 
   return (
-    <div ref={container} id={id} className="split" onMouseMove={handleMouseMove}>
+    <div ref={container} id={id} className={`split ${className}`} onMouseMove={handleMouseMove}>
```

Comment:
Note to myself to look where this is used.

Reply:
using it to add the `fade-in`/`out` class

Reply:
(bc it needs to be applied to both the slide view + the inspector)